config/betterquesting/DefaultQuests/Quests/MultiblockGoals-AAAAAAAAAAAAAAAAAAAADA==/EBFTime-AAAAAAAAAAAAAAAAAAAATQ==.json

```diff
@@ -31,7 +31,7 @@
+      "desc:8": "In order to get alumin(i)um ingots, you need an Electric Blast Furnace. It\u0027s another very important step towards victory, so let\u0027s get started.\n\nGetting the Mica for the heating coils might be a little tricky, but you can use a substitute for the first tier of coils. \nYou need 2 LV hatches with 4A of 32V total, or 3 LV hatches with MOOOORE power if the multiblock is not fully repaired. \nIn order to not scrape by for Nickel, check out the quest Source of Nickel in the Novice Thaumaturgy tab.\n\n§rA few recipes produce CO2/SO2 as a fluid. You have put an output fluid hatch to the top layer of the EBF to collect it otherwise the EBF won\u0027t start. It depends on the muffler hatch how much fluid you can recover. Bart\u0027s Platinum Series recipes output fluids on the BOTTOM, so if you want to do Platinum Stuff later, remember that. If you want to void fluids anyway, just enable fluid voiding in the controller.\n\nMoron always says - Make Muffler Moint Mup. Or point up, anyways. Before the launch, don\u0027t forget to fix maintenance issues and put a programmed circuit if necessary.\n\nIf you want a really fancy EBF, you can add more input fluid hatches and store different fluids in each one.\n\nThe EBF also has a heat adjustment of 100K for each power tier above or below MV. You can only make LV energy hatches right now, but installing two LV energy hatches will overclock the EBF to MV power, eliminating the 100K penalty and setting the initial heat to 1801K.\n\nAs you upgrade your EBF over the course of the game, this heat adjustment will increase. Two MV energy hatches (overclocking to HV) will add +100K, two HV energy hatches (overclocking to EV) will give +200K, and so on. Coils only set the base value. This adjustment does not apply to the Volcanus.\n\nIf the pollution gets really bad, check in GT++ Multiblocks for a pollution scrubber, or upgrade your Muffler.\n"
```

Comment:
"You have to put an output"

I would reword it slightly so that it is more like:

 You have to put an output fluid hatch to the top layer of the EBF to collect it, or enable fluid voiding, otherwise the EBF won\u0027t start.

and remove the later line. Does EBF ever make good fluids?

Reply:
I never used those fluids, but I can see why someone would want to recapture them (sulfur dioxide for example for sulfuric acid).